SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -109,10 +109,13 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
     }
 
     @IBAction func placeCall(_ sender: UIButton) {
```

Comment:
Let's rename the method to `mainButtonPressed` as it turns into `disconnect` and performs disconnect when there is a live Call. Same applies to `placeCallButton`.